src/devices_models/devices/storage.jl

```diff
@@ -47,7 +47,7 @@ function DeviceRangeConstraintSpec(
     ::Type{<:RangeConstraint},
     ::Type{ActivePowerOutVariable},
     ::Type{T},
-    ::Type{<:AbstractStorageFormulation},
+    ::Type{<:BookKeeping},
```

Comment:
We need to rename BookKeeping since in principle both do bookkeeping one just happens to have reservation variables.